### ISSUE TITLE
Errors in ValidationErrorResponse class

### ISSUE THREAD

**Issue by serggl** [phrase/phraseapp-ruby#11]
current version if this class looks like this:
```
class ValidationErrorResponse
    	attr_accessor :error_response
      attr_accessor :errors

      def initialize(http_response)
        hash = JSON.load(http_response.body)
        puts hash.inspect
      end

      def errors
        self.error + "\n" + self.errors.join("\n")
      end
    end
```

2 issues here:
1) `error_response` is never set in constructor
2) calling `errors` on this class instance will raise error, since referred `error` method does not exist

**Comment by fredv** [comment]
Hi @serggl, thank you for pointing this out. Looks like a left-over from our auto-generation. We will clean this up.

**Comment by kennell** [comment]
Hi @serggl, we just released version 1.2.3 that fixes some issues in the error responses classes.
